WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -126,11 +126,29 @@ public function getGroups() {
 				),
 			),
 
+			'unlink' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use wp_delete_file() for remote URLs instead.',
+				'since'     => 'x.x.x',
```

Comment:
Looking at the docs, the `wp_delete_file()`function was added in WP 4.2.0, so I believe this should be set to `4.2.0`.
Ref: https://developer.wordpress.org/reference/functions/wp_delete_file/

Reply:
Added